packages/main/src/types/TableCellAlign.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Alignment of the &lt;ui5-table-cell&gt; component.
+ *
+ * @public
+ */
+enum TableCellAlign {
+    /**
+     * @public
+     */
+    Left = "left",
```

Comment:
What is written into the DOM? Is it "Right" or "right"? AFAIK it should be "Right", as we never convert the given String to the actual Enum value.

Not a big deal, because it seems to work fine uppercase, but not sure if this could lead to issues with certain browsers or validators?
Is CSS case-sensitive?

Maybe we need to a) use the actual enum values (which would require us to do some sort of validation), b) change the enum names to lower case.

@aborjinik What's your opinion on this?

Reply:
Even after changing the enum definition to use "Left" instead of "left" and changing the example.html using h-Align="Left" it's still lowercase.

Reply:
I've changed it to capital letters.
As discussed I restrict the values of hAlign to the ones in TableCellHorizontalAlign.